backend/rust/bunker/build.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    tonic_build::configure()
+        .out_dir("src")
```

Comment:
It is better to not put the autogenerated code in `src/`.

Reply:
Really good suggestion. It was already moved to the `bunker/generated` folder.

Reply:
And `codegen` mod was added only for creating backend.rs